mysql-test/r/percona_dd_upgrade_8013.result

```diff
@@ -19,10 +19,10 @@ se_private_id
 SET SESSION DEBUG = '+d,skip_dd_table_access_check';
 SELECT se_private_id FROM mysql.tables WHERE name = 'compression_dictionary';
 se_private_id
-34
+2120
 SELECT se_private_id FROM mysql.tables WHERE name = 'compression_dictionary_cols';
 se_private_id
-35
+2121
 CREATE COMPRESSION_DICTIONARY d1('blah');
```

Comment:
@dutow @kamil-holubicki this test is no longer exists after Kamils PR
and also this result change means, there is an issue :) 

I should have use assert.inc instead of simple SELECT :)

Reply:
Upstream MTR tests all replace se_private_id in the output with NNN or something similar with a generic regex. As it wasn't causing a crash in PS, it made sense to me that an upstream change could mean that we are using a different id from now on.